mozilla-mobile/focus-android/app/src/main/res/values/strings.xml

```diff
@@ -550,7 +547,7 @@ be temporary, and you can try again later.</li>
     <!-- First run tour (Shortcut): Title -->
     <string name="firstrun_shortcut_title">Add shortcuts to your home screen</string>
     <!-- First run tour (Shortcut): Text. %1$s will be replaced with the name of the app (e.g. Firefox Focus) -->
-    <string name="firstrun_shortcut_text">Return to your favorite sites in %1$s quickly. Just select \"Add to Home screen\" from the %1$s menu.</string>
+    <string name="firstrun_shortcut_text">Return to your favorite sites in %1$s quickly. Just select \&quot;Add to Home screen\&quot; from the %1$s menu.</string>
```

Comment:
Oh, this looks like it created an error. We need to fix that string in the Focus repository.

Reply:
Double-checking with Fenix this actually seems to be correct. In the app repositories it's `\"` and there in this repository the tool uses `\&quot;` (e.g. see `preference_summary_delete_browsing_data_on_quit_2` in Fenix).

Reply:
I wonder if `&quot;` would be sufficient here? 🤷

We have both, `&quot;` and `\&quot;` in Fenix (which is a surprise of it's own 😃):
https://github.com/mozilla-l10n/android-l10n/blob/master/mozilla-mobile/fenix/app/src/main/res/values/strings.xml#L1097-L1100